### PR TITLE
add incremental embedding with table notebook

### PR DESCRIPTION
**Pull request by sachinparyani**
# Description
Added a notebook that customers can reference to utilize the incremental embedding feature and also see how they can use unstructured files extensions like .docx

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/python/generative-ai/rag/notebooks/faiss/url_to_faiss_incremental_embeddings_with_tabular_data.ipynb [hunk 1]
```diff
@@ -0,0 +1,963 @@
...
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "asset_name = 'microsoft-earnings-fy23'\n",
+        "data_source_glob = '**/*[Q1]_10Q.docx'\n",
+        "\n",
+        "pipeline_job = urifolder_to_faiss(\n",
+        "    input_data=Input(type=AssetTypes.URI_FOLDER, path=\"azureml://datastores/workspaceblobstore/paths/msft_earnings_fy23\"),\n",
```
Comment (tot0) [inline sdk/python/generative-ai/rag/notebooks/faiss/url_to_faiss_incremental_embeddings_with_tabular_data.ipynb:556]: This requires users to upload the test data to this location on their workspaceblobstore. Take a look at `acs_mlindex_with_langchain.ipynb` for an example of using a local path as data input, the data will automatically be uploaded for the user.

File: sdk/python/generative-ai/rag/notebooks/faiss/url_to_faiss_incremental_embeddings_with_tabular_data.ipynb [hunk 2]
```diff
@@ -0,0 +1,963 @@
...
+          "execution_count": 49,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "retriever = MLIndex(ml_client.data.get(asset_name, label='latest')).as_langchain_retriever()\n",
+        "qa = RetrievalQA.from_chain_type(llm=init_llm(model_config), chain_type=\"stuff\", retriever=retriever)\n",
+        "question = \"In the three months ended September 30, 2022, what were the expenses of Microsoft in the 'General and administrative' segment and as a percentage of the revenue?\"\n",
+        "retriever.get_relevant_documents(question)\n",
```
Comment (tot0) [inline sdk/python/generative-ai/rag/notebooks/faiss/url_to_faiss_incremental_embeddings_with_tabular_data.ipynb:741]: The return value of this call aren't used/printed, should it be removed?

**Review by tot0 — CHANGES_REQUESTED** [review]
Overall looks good!

**Review by balapv — APPROVED** [review]
This is approved from my end if @tot0 is ok with this file

**Review by tot0 — APPROVED** [review]

**Pull request merged**